.docker/tests/conftest.py

```diff
@@ -3,21 +3,33 @@
 
 import pytest
 
+TARGETS = ('aiida-core-base', 'aiida-core-with-services', 'aiida-core-dev')
 
-@pytest.fixture(
-    scope='session',
-    params=[
-        'aiida-core-base',
-        'aiida-core-with-services',
-        'aiida-core-dev',
-    ],
-)
-def variant(request):
-    return request.param
+
+def target_checker(value):
+    msg = f"Invalid image target '{value}', must be one of: {TARGETS}"
+    if value not in TARGETS:
+        raise pytest.UsageError(msg)
+    return value
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--variant',
```

Comment:
It would be better to use `--target` here to be consistent with `aiidalab-docker-stack` but I'll do that in a separate PR.